hadoop-hdds/common/src/main/conf/ozone-env.sh

```diff
@@ -0,0 +1,280 @@
+
+# The maximum amount of heap to use (Java -Xmx).  If no unit
+# is provided, it will be converted to MB.  Daemons will
+# prefer any Xmx setting in their respective _OPT variable.
+# There is no default; the JVM will autoscale based upon machine
+# memory size.
+# export OZONE_HEAPSIZE_MAX=
+
+# The minimum amount of heap to use (Java -Xms).  If no unit
+# is provided, it will be converted to MB.  Daemons will
+# prefer any Xms setting in their respective _OPT variable.
+# There is no default; the JVM will autoscale based upon machine
+# memory size.
+# export OZONE_HEAPSIZE_MIN=
+
+# Extra Java runtime options for all Ozone commands. We don't support
+# IPv6 yet/still, so by default the preference is set to IPv4.
+# export OZONE_OPTS="-Djava.net.preferIPv4Stack=true"
+# For Kerberos debugging, an extended option set logs more information
+# export OZONE_OPTS="-Djava.net.preferIPv4Stack=true -Dsun.security.krb5.debug=true -Dsun.security.spnego.debug"
+
+# Some parts of the shell code may do special things dependent upon
+# the operating system.  We have to set this here. See the next
+# section as to why....
+export OZONE_OS_TYPE=${OZONE_OS_TYPE:-$(uname -s)}
+
+# Extra Java runtime options for some Ozone commands
+# and clients (i.e., hdfs dfs -blah).  These get appended to OZONE_OPTS for
+# such commands.  In most cases, # this should be left empty and
+# let users supply it on the command line.
+# export OZONE_CLIENT_OPTS=""
+
+#
+# A note about classpaths.
+#
+# By default, Apache Ozone overrides Java's CLASSPATH
+# environment variable.  It is configured such
+# that it starts out blank with new entries added after passing
+# a series of checks (file/dir exists, not already listed aka
+# de-deduplication).  During de-deduplication, wildcards and/or
+# directories are *NOT* expanded to keep it simple. Therefore,
+# if the computed classpath has two specific mentions of
+# awesome-methods-1.0.jar, only the first one added will be seen.
+# If two directories are in the classpath that both contain
+# awesome-methods-1.0.jar, then Java will pick up both versions.
+
+# An additional, custom CLASSPATH. Site-wide configs should be
+# handled via the shellprofile functionality, utilizing the
+# ozone_add_classpath function for greater control and much
+# harder for apps/end-users to accidentally override.
+# Similarly, end users should utilize ${HOME}/.ozonerc .
+# This variable should ideally only be used as a short-cut,
+# interactive way for temporary additions on the command line.
+# export OZONE_CLASSPATH="/some/cool/path/on/your/machine"
+
+# Should OZONE_CLASSPATH be first in the official CLASSPATH?
+# export OZONE_USER_CLASSPATH_FIRST="yes"
+
+# If OZONE_USE_CLIENT_CLASSLOADER is set, OZONE_CLASSPATH and
+# OZONE_USER_CLASSPATH_FIRST are ignored.
+# export OZONE_USE_CLIENT_CLASSLOADER=true
+
+###
+# Options for remote shell connectivity
+###
+
+# There are some optional components of hadoop that allow for
```

Comment:
start-dfs.sh is mentioned two times here, can you please rephrase this comment, and the next which mentions it to point to start-ozone.sh, and to mention Ozone roles?

Reply:
Thanks.